lua/astrocommunity/pack/rust/README.md

```diff
@@ -42,3 +42,31 @@ return {
   },
 }
 ```
+
+## Use `cargo check` for `rust-analyzer.check.command`
+
+By default, this pack configures `rust-analyzer.check.command` to use `cargo clippy` instead of `cargo check`.
+
+In some cases (for example, on large projects where `cargo clippy` is slow), you may want to use `cargo check` instead.
+
+There is currently no way to configure `rust-analyzer` per project, [adding `rust-analyzer.toml` support is still in progress](https://github.com/rust-lang/rust-analyzer/issues/13529).
+If you want to use `cargo check` you can change the `rust-analyzer.check.command` globally like the following:
+
```

Comment:
https://github.com/mrcjkb/rustaceanvim , which we're  providing actually supports this.

Reply:
lol, I didn't know that. Updated.